scripts/pre_push_hook.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python2.7
+"""Pre-push hook that executes the Python/JS linters on all files that
+deviate from develop.
+(By providing the list of files to `scripts/pre_commit_linter.py`)
+To install the hook manually simply execute this script with the `--install`
+flag.
+"""
+
+import os
+import sys
+import subprocess
+import collections
+import pprint
+import argparse
+import shutil
+
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument("remote", nargs="?", help="provided by git before push")
```

Comment:
Bit of a nit, but would you mind using single quotes for strings throughout this file? We do this by convention throughout the codebase.